rfc/new-artifacts-service/README.md

```diff
@@ -0,0 +1,207 @@
+# Separate artifacts to new service
+
+| | |
+|-|-|
+| **Status**     | **Proposed**, Accepted, Implemented, Obsolete |
+| **RFC #**      | _Update after PR has been made_ |
+| **Author(s)**  | [Nemesis Osorio](https://github.com/nemesisOsorio), [Sergio Quintero](https://github.com/sergio-quintero) |
+| **SIG / WG**   | _Applicable SIG(s) or Working Group(s)_ |
+
+## Overview
+
+Users with large context pipelines do not have a good experience with pipeline executions. <br/>
+Context and outputs in pipeline executions can contain large artifacts/objects such as k8s manifest (yaml/json files). <br/>
+Spinnaker passes these artifacts/objects in request and response bodies across some services. <br/>
+The idea is to create a new service to store these artifacts/objects
```

Comment:
Do we want a service initially?  OR just mask this in the APIs returning pipeline data to gate?

Reply:
If we create a new service, we can reuse the produced artifact across different services.
For example: Bake and Deploy a Kustomize template: 
1. Rosco produces and saves the artifact and returns the id to Orca 
2. Orca sends the deploy operation to Clouddriver using the artifact id
3. Clouddriver gets the artifact by id

I believe your second question is covered in the `Alternatives` section.